doc/api/assert.md

```diff
@@ -33,13 +33,16 @@ added: v0.1.21
 -->
 
 Tests for deep equality between the `actual` and `expected` parameters.
-Primitive values are compared with the equal comparison operator ( `==` ).
+Primitive values are compared with the [Abstract Equality Comparison][]
+( `==` ).
 
-Only enumerable "own" properties are considered. The `deepEqual()`
-implementation does not test object prototypes, attached symbols, or
-non-enumerable properties. This can lead to some potentially surprising
-results. For example, the following example does not throw an `AssertionError`
-because the properties on the [`Error`][] object are non-enumerable:
+Only [enumerable "own" properties][] are considered. The `deepEqual()`
+implementation does not test the [`[[Prototype]]`][prototype-spec] of
+objects, attached symbols, or non-enumerable properties. (For those checks,
```

Comment:
Just spotted this... can you actually remove the parens in this sentence. It can be reworded slightly as:

```
Only [enumerable "own" properties][] are considered. The `deepEqual()`
implementation does not test the [`[[Prototype]]`][prototype-spec] of objects,
attached symbols, or non-enumerable properties — for such checks, consider
using [assert.deepStrictEqual()][] instead.
```

Also, using `deepEqual()` in one sentence and `assert.deepStrictEqual()` in another is a bit inconsistent. Consider either using the `assert.` in both or omitting it from both.